equalitie/ceno
# Portal firefox

Has the Firefox extension set the new tab page and the homepage of the user's browser to the CENO portal while active.

To test:

1. Open a clean Firefox profile
2. Start the CENO client
3. Install the extension by opening the included cenorouter version 1.1.0 file in Firefox
4. Check that the CENO Router is active; if not, activate it
5. Open a new tab and verify it is the CENO Portal
6. Navigate to another site
7. Press the home button and verify that firefox goes to the CENO Portal
